src/parse.js

```diff
@@ -220,10 +220,8 @@ export const getTraverser = (cb = noop, opts = {}) => {
         const envOpts = state.opts || opts;
         const propsMap = envOpts.componentPropsMap || GETTEXT_COMPONENT_PROPS_MAP;
 
-        // Using fragment short syntax <>...</> yields JSXText paths with
-        // whitespace content and no name property, so this check is best
-        // done before invoking isGettextComponent().
-        if (node.value.trim() === '') {
+        // parent.openingElement can be undefined when using fragment short syntax <>...</>
+        if (!parent.openingElement) {
```

Comment:
I'm curious how you came up with this check?

Reply:
I found this just by trial and error :)
But after that I found that JSX fragment has `openingFragment` rather than `openingElement`
https://github.com/facebook/jsx/blob/master/AST.md#jsx-fragment